dockerfiles/run/guice/cassandra-ldap/destination/conf/batchsizes.properties

```diff
@@ -1,4 +1,6 @@
-metadata=200
-headers=200
-body=100
-full=50
\ No newline at end of file
+fetch.metadata=200
```

Comment:
Just wonder: as an user, example the system admin guy who do not know about james and sometimes he has to change those values, like the other properties files in the conf folder. 
Then it should be confuse for them if they don't know what is the purpose of those values. Could we have some description (or comment) on each of those values?